packages/utils-debugging-protocol-common/src/debugging-protocol-connector.ts

```diff
@@ -706,13 +707,13 @@ export class Connector implements IConnector {
         const { Network } = this._client;
 
         await Promise.all([
-            Network.clearBrowserCache(),
-            Network.setCacheDisabled({ cacheDisabled: true }),
+            Network.clearBrowserCache!(),
+            Network.setCacheDisabled!({ cacheDisabled: true }),
             // The typings we use for CDP aren't 100% compatible with our libarary
-            Network['requestWillBeSent'](this.onRequestWillBeSent.bind(this)), // eslint-disable-line dot-notation
-            Network['responseReceived'](this.onResponseReceived.bind(this)), // eslint-disable-line dot-notation
-            Network['loadingFinished'](this.onLoadingFinished.bind(this)), // eslint-disable-line dot-notation
-            Network['loadingFailed'](this.onLoadingFailed.bind(this)) // eslint-disable-line dot-notation
+            (Network as any)['requestWillBeSent'](this.onRequestWillBeSent.bind(this)), // eslint-disable-line dot-notation
```

Comment:
Why are we using ` (Network as any)['requestWillBeSent'](this.onRequestWillBeSent.bind(this))` instead of `Network.requestWillBeSent`?

Reply:
Because the types we have aren't 100% compatible with the library we are using for CDP. The types provided by the chrome team use `onRequestWillBeSent` but [`chrome-remote-interface`](https://github.com/cyrus-and/chrome-remote-interface) has them as `requestWillbeSent`.

Reply:
ok!